apisix/init.lua

```diff
@@ -260,8 +260,8 @@ local function set_upstream_host(api_ctx, picked_server)
     if nodes_count == 1 then
         local node = up_conf.nodes[1]
         api_ctx.var.upstream_host = node.domain or node.host
```

Comment:
We should also handle this path (nodes_count == 1)

Reply:
Sorry, I have been busy with work and have no time to reply.ok, I'll deal with it later.